microsite/data/plugins/azure-resource.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: 'azure-resource'
+author: 'Anurag Pachauri'
+authorUrl: 'https://github.com/anuragpachauri'
+category: Infrastructure
+description: 'View resources based of tag and value provided in the catalog-info.yaml file.'
+documentation: 'https://github.com/anuragpachauri/azure-resource/blob/main/README.md'
```

Comment:
Thanks! I can't seem to find the sourcecode 🤔 
Perhaps the url is wrong? https://github.com/anuragpachauri/azure-resource